sssd_test_framework/hosts/ldap.py

```diff
@@ -35,6 +39,22 @@ def __init__(self, *args, **kwargs) -> None:
         # Backup of original data
         self.__backup: dict[str, dict[str, list[bytes]]] = {}
 
+    def pytest_setup(self) -> None:
+        super().pytest_setup()
+        # Make sure that ldap service is enabled
+        enabled = self.ssh.run(f"systemctl is-enabled {self._ldap_service_name}")
```

Comment:
Why? This only makes the service start automatically after reboot. It doesn't affect manual start/stop, does it?

Reply:
It was a defensive way of making sure that when some test (for some reason) restarts the machine, it will come back up.  Removing it now as I double checked that sssd-ci-conmtainers should be doing that.